cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -465,13 +469,23 @@ func (s *Server) Index(args *indexArgs) (state indexState, err error) {
 		return indexStateEmpty, createEmptyShard(args)
 	}
 
+	repositoryName := args.Name
+	if _, ok := s.deltaBuildRepositoriesAllowList[repositoryName]; ok {
+		repositoryID := args.BuildOptions().RepositoryDescription.ID
+		debug.Printf("(delta build) Server.Index: marking %q (ID %d) for delta build", repositoryName, repositoryID)
```

Comment:
This style of `(delta build)` is not something I've normally seen in go code. More common is something is like `delta: Server.Index:`. Additionally you already mention delta in the log, so it's already greppable with just delta. Another minor reason against using parenthesis as a specifier is they are slightly less ergonomic to grep for since ( is a special char in regex.